mysql-test/suite/galera/t/galera_wait_for_executed_gtid_set.test

```diff
@@ -2,15 +2,11 @@
 # Test the WAIT_FOR_EXECUTED_GTID_SET() function
 #
 
---source include/have_log_bin.inc
---source include/galera_cluster.inc
---source include/have_innodb.inc
-<<<<<<< HEAD
 --source include/have_debug.inc
```

Comment:
do we still need have_debug.inc?

This question is relevant to other *.test files as well

Reply:
Since our Jenkins infrastructure always builds both the server and galera in same mode (galera is buillt in debug iff server is also built in debug). So, not having `have_debug.inc` may cause the tests to fail with below error.
```
CURRENT_TEST: galera.galera_wait_for_executed_gtid_set
mysqltest: In included file ./include/galera_set_sync_point.inc at line 2:
included from /home/venki/work/pxc/3352/57/mysql-test/suite/galera/t/galera_wait_for_executed_gtid_set.test at line 18:
At line 1: query 'SET GLOBAL wsrep_provider_options = 'dbug=d,$galera_sync_point'' failed: 1210: Incorrect arguments to SET

The result from queries just before the failure was:
CREATE TABLE t1 (f1 INT PRIMARY KEY) ENGINE=InnoDB;

```